.github/workflows/continuous-integration.yml

```
@@ -23,28 +23,18 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "7.1"
-          - "7.2"
-          - "7.3"
           - "7.4"
           - "8.0"
           - "8.1"
           - "8.2"
+          - "8.3"
         dependencies:
           - "highest"
         stability:
           - "stable"
-        extra_constraint:
-          - ""
         include:
           - dependencies: "lowest"
-            php-version: "7.1"
-          - dependencies: "highest"
-            extra_constraint: "symfony/lts:v3"
-            php-version: "7.3"
-          - dependencies: "highest"
-            stability: "dev"
-            php-version: "7.3"
+            php-version: "7.4"
           - dependencies: "highest"
             stability: "dev"
             php-version: "8.2"
```

Isn't this supposed to be 8.3? Or maybe a

```suggestion
            php-version: "8.2"
          - dependencies: "highest"
            stability: "dev"
            php-version: "8.3"
```

is needed?

---

Yes. I've done this in #405 now.